pysatModels/models/ucar_tiegcm.py

```diff
@@ -182,8 +203,14 @@ def load(fnames, tag='', inst_id='', **kwargs):
     return data, meta
 
 
+# Set the download routine
+basic_tag = {'remote_dir': '/pub/data/icon/l4/tiegcm/{year:04d}/',
+             'fname': remote_fname}
+download_tags = {'': {'icon': basic_tag}}
```

Comment:
Why are these being set outside of the download function?

Reply:
Also used by the `list_remote_files` function